enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -22,67 +22,67 @@ msgstr ""
 msgid "Unlink selected objects from existing course catalogs"
 msgstr ""
 
-#: enterprise/admin/forms.py:29
+#: enterprise/admin/forms.py:30
 msgid "Type in Email or Username to link single user"
 msgstr ""
 
-#: enterprise/admin/forms.py:31
+#: enterprise/admin/forms.py:32
 msgid "Or upload a CSV to enroll multiple users at once"
 msgstr ""
 
-#: enterprise/admin/forms.py:32
+#: enterprise/admin/forms.py:33
 msgid ""
 "The CSV must have a column of email addresses, indicated by the heading "
 "'email' in the first row."
 msgstr ""
 
-#: enterprise/admin/forms.py:35
+#: enterprise/admin/forms.py:36
 msgid "Also enroll these learners in this course"
 msgstr ""
 
-#: enterprise/admin/forms.py:36
+#: enterprise/admin/forms.py:37
 msgid "Provide a course ID if enrollment is desired."
 msgstr ""
 
-#: enterprise/admin/forms.py:39
+#: enterprise/admin/forms.py:40
 msgid "Course enrollment mode"
 msgstr ""
 
-#: enterprise/admin/forms.py:41
+#: enterprise/admin/forms.py:42
 msgid "Audit"
 msgstr ""
 
-#: enterprise/admin/forms.py:42
+#: enterprise/admin/forms.py:43
 msgid "Verified"
 msgstr ""
 
-#: enterprise/admin/forms.py:43
+#: enterprise/admin/forms.py:44
 msgid "Professional Education"
 msgstr ""
 
-#: enterprise/admin/forms.py:44
+#: enterprise/admin/forms.py:45
 msgid "Professional Education (no ID)"
 msgstr ""
 
-#: enterprise/admin/forms.py:45
+#: enterprise/admin/forms.py:46
 msgid "Credit"
 msgstr ""
 
-#: enterprise/admin/forms.py:46
+#: enterprise/admin/forms.py:47
 msgid "Honor"
 msgstr ""
 
-#: enterprise/admin/forms.py:192
+#: enterprise/admin/forms.py:208
 msgid "None"
 msgstr ""
 
-#: enterprise/admin/forms.py:240
+#: enterprise/admin/forms.py:256
 msgid ""
 "Selected Identity Provider does not exist, please contact system "
 "administrator for more info."
 msgstr ""
 
-#: enterprise/admin/forms.py:250
+#: enterprise/admin/forms.py:266
 #, python-brace-format
 msgid ""
```

Comment:
Since GH won't let me comment on lines 89-148, I'm adding message suggestions for all of these here.

**ll. 88-91**
The site for the selected identity provider ({identity_provider_site}) does not match the site for this enterprise customer ({enterprise_customer_site}). To correct this problem, select a site that has a domain of '{identity_provider_site}', or update the site for the identity provider to '{enterprise_customer_site}' <where does the user do this? e.g., on the Identity Provider page>.

 **ll. 96-97**
Specify either one email address or username to add, or one .csv file to upload.

**ll. 102-103**
Error: Learners could not be added. Take the following action to correct the error.

**l. 108**
Select a valid enrollment track for course {course_id}.

**l. 112**
Select a valid enrollment track for course {course_id}.

**ll. 118-119**
Specify a valid course ID.

**l. 124**
Specify a valid email address.

**l. 129**
Specify a valid email address or username.

**ll. 135-136**
Verify that the .csv file has a column named {name}.

**ll. 141-142**
Specify either one email address or username to add, or one .csv file to upload.

**ll. 148-149**
A learner with the {email} address is already registered with enterprise customer {ec_name}. Enter a different email address.